File: project/angular_project/src/app/models/review.ts
import { Employee } from "./employee";

export class Review {
  review_id: number;
  performance_rating: number;
  areas_of_improvement: string;
  title: string;
  employee: Employee;

  constructor() {
    this.review_id = 0;
    this.performance_rating = 0;
    this.areas_of_improvement = '';
    this.title = '';
    this.employee = new Employee(); 
  }
}

File: src/main/java/com/example/project/entities/Review.java
package com.example.project.entities;

import java.io.Serializable;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.DBRef;
import org.springframework.data.mongodb.core.mapping.Document;

import lombok.Data;


@Data
@Document(collection = "Review")
public class Review implements Serializable {
	@Id
	private int review_id;
	private int performance_rating;
	private String areas_of_improvement;
	private String title;
	//reviewed by
    private Employee employee;
	
}